tests/cpp/test_sdpa_node.cpp

```diff
@@ -188,44 +223,54 @@ TEST_F(SDPATest, PairwiseRootDomainMap) {
   fusion->addInput(tvk);
   fusion->addInput(tvv);
 
-  auto tvattn = sdpfa_fwd(
+  auto output = sdpfa_fwd(
       tvq,
       tvk,
       tvv,
       /*dropout_p=*/IrBuilder::create<Val>(0.0),
       /*is_causal=*/IrBuilder::create<Val>(true),
       /*scale=*/IrBuilder::create<Val>(1e-3));
-  fusion->addOutput(tvattn.output);
-
-  // Verify mapping between Q,K,V and attention output
+  addSdpaFwdOutputs(fusion.get(), output);
+
+  // Verify mapping between Q,K,V and all output
+  // Producers:
+  //   query = [N, H, L, E]
+  //   key = [N, H, S, E]
+  //   value = [N, H, S, Ev]
```

Comment:
nitpick: Looks like we support broadcasting on the batch dimension from the code changes above? do we actually have a test case for that where one of the entries have a broadcast batch dimension?

Reply:
Yes, broadcast is allowed on N, H.
I have a separate branch adding some checks enforcing `E = Ev` and the corresponding mapping, I will include it there.

Thanks for bringing this up.

Reply:
Feel free to add that in a separate PR if that works better~